client/package.json

```diff
@@ -62,9 +64,9 @@
     "react-dom": "^15.4.2",
     "react-tether": "^0.5.5",
     "run-sequence": "^1.2.2",
-    "showdown": "^1.9.0",
-    "@turf/turf": "5.x",
-    "ng-table": "^3.0.1"
+    "showdown": "^1.9.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "node": "^11.15.0"
```

Comment:
This is (pretty much) a deal breaker right now.

The version of node on the server is v8. While this PR _might_ work on the server, it would be appreciated if you could test with the server version (the way node was installed on the server makes it a PITA to update).